### DISCUSSION TITLE
Loading scenes or robot files for MJCF descriptions

### DISCUSSION THREAD

**Discussion by stephane-caron** [robot-descriptions/robot_descriptions.py#79] [Design]
When loading a robot description from MuJoCo Menagerie, it seens it is more common to load the `scene.xml` files for each description rather than the individual robot files (that have, notably, no ground).

The proposal is then to make all MJCF descriptions loaded via `robot_descriptions` go for `scene.xml` by default, with optionally a variant https://github.com/robot-descriptions/robot_descriptions.py/issues/62 for bare robots.

**Comment by stephane-caron** [comment]
@kevinzakka Do you think this would make sense for the majority of users?

I can take care of making a PR for this, sometime in a not so-long time.

**Comment by lvjonok** [comment]
Let me add something from my experience of working with MJCF models. I, personally, didn't find many reasons behind holding very similar `scene.xml` from each Mujoco model and, thus, developed a tool, which automatically generated it given `z` position of floor intended.

I think a similar solution will be even better in long-term for general-purpose usage

**Comment by kevinzakka** [comment]
In Menagerie, the `scene.xml` is mostly there to have consistent plane, lighting, free-camera settings and for overall aesthetics. I think it's more proper to load the canonical model and have a separate `scene` variant. In general, the variant option would be very useful :)

**Comment by stephane-caron** [comment]
Thank you both for your feedback! Then it seems the best way forward is to allow it but not make it the default.
